src/tools/Repl/Program.cs

```diff
@@ -617,6 +645,12 @@ public FormulaValue Execute(StringValue option, BooleanValue value)
                     return value;
                 }
 
+                if (option.Value.ToLower(CultureInfo.InvariantCulture) == OptionHashCodes.ToLower(CultureInfo.InvariantCulture))
```

Comment:
>ToLower(CultureInfo.InvariantCulture) [](http://example.com/codeflow?start=33&length=37)

nit: ToLowerInvariant

Reply:
Done.

Reply:
sorry, realized there's an aequality here. You could even just say:
String.Equals(a,b,StringComparison.OrdinalIgnoreCase)